introduction.md

```diff
@@ -1 +1,2 @@
-I love Ruby and JavaScript!
\ No newline at end of file
+I love Ruby and JavaScript!
+* Ruby, JS, git
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?